openssl/src/ssl/mod.rs

```diff
@@ -2394,6 +2404,14 @@ impl SslRef {
         unsafe { ffi::SSL_set_verify(self.as_ptr(), mode.bits as c_int, None) }
     }
 
+    /// Returns the verify mode that was set using `set_verify`.
+    ///
+    /// This corresponds to `SSL_get_verify_mode`.
+    pub fn verify_mode(&self) -> SslVerifyMode {
```

Comment:
I called this `verify_mode` even though the setter is just `set_verify` because I thought that a function called `verify()` would seem like it does some kind of certificate verification.

Reply:
Seems fine to me, and matches the raw OpenSSL name.

Reply:
Can you add links to the man page for this and is_init_finished below?